akka-persistence-tck/src/main/scala/akka/persistence/CapabilityFlags.scala

```diff
@@ -52,3 +52,9 @@ trait SnapshotStoreCapabilityFlags extends CapabilityFlags {
   // no flags currently
 }
 //#snapshot-store-flags
+
+trait CurrentEventsByTagCapabilityFlags extends CapabilityFlags {
```

Comment:
@patriknw Which capabilities should be available in the *byTag queries? I came up with:
- `supportsOrderingByDateIndependentlyOfPersistenceId ` for eg. JDBC storage engines,
- `supportsOrderingByPersistenceIdAndSequenceNr` for eg. Cassandra, Redis, Riak?